main.js

```diff
@@ -38,9 +38,18 @@ var mantras = [
 "Onward and upward.",
 "I am the sky, the rest is weather.",
 ];
-var favorites = [];
-
-
+var localStorageFavorites = window.localStorage.getItem("favorites");
+var parsedFavorites = JSON.parse(localStorageFavorites);
+//if parsedFavorites <--looking for truthy, so if parsedFavorites contains anything, it is truthy
+  //? true
+  //: false
+var favorites = (parsedFavorites.length)
```

Comment:
So, I ran into a little bug when I tried this out! I think we can replicate it on your machine by completely clearing `localStorage` (by running `localStorage.clear()` in the dev tools) and then refreshing the window. When there is no `favorites` array in `localStorage`, the `getItem` on line 41 returns `null`. Therefore, `parsedFavorites` is assigned to `null`.

This means that it doesn't have a `.length` property and crashes your app for users who haven't visited before. Based on my exploration, if we change `parsedFavorites.length` to `parsedFavorites`, this won't crash the app. This still works with your ternary statement because `null` is `false`-y (it just doesn't have a length property)! 

Feel free to let me know if this isn't coming across! Happy to jump on a call and talk about it!